t/t0001-basic.t

```diff
@@ -310,6 +310,15 @@ test_expect_success 'flux-start in exec mode works as initial program' "
 test_expect_success 'flux-start in subprocess/pmi mode works as initial program' "
 	flux start ${ARGS} -s2 flux start ${ARGS} -s1 flux getattr size | grep -x 1
 "
+# See issue #5447
+test_expect_success 'flux-start works with non-errexit clean BASH_ENV' '
+	cat >testbashrc <<-EOT &&
+	command -v ls >/dev/null || :
+	/bin/false
+	/bin/true
```

Comment:
perhaps I'm missing something, but what is the purpose of the `command -v ls` line.  Shouldn't /bin/false & /bin/true be enough?  Or just wanted to stick something in randomly before the /bin/false?

Reply:
It is discussed in the original issue #5447, but I agree that is esoteric enough it could use a comment.

Reply:
OK I'll add something